code/service/src/main/scala/com/github/chiefofstate/readside/ReadSideBootstrap.scala

```diff
@@ -55,13 +57,22 @@ class ReadSideBootstrap(
         new ReadSideProjection(system, config.readSideId, dataSource, remoteReadSideProcessor, numShards)
       // start the sharded daemon process
       projection.start()
+
+      // pause readSide for all shards after starting it if pause on start is enable
+      if (config.pausedOnStart) {
+        readSideManager.pauseForAll(config.readSideId)
+      }
```

Comment:
I think they wanted it to only start paused when it is created. I believe this would always start paused on boot

Reply:
Before we can pause the Akka projection it must be started. So with this when the setting is enabled then the projection is paused after started. Yeah I see the caveat here, the projection may have started processing some message in the time frame when it is getting paused. Let me think through this a bit.